src/L.Control.Heightgraph.js

```diff
@@ -123,6 +136,10 @@ L.Control.Heightgraph = L.Control.extend({
             this._dragRectangleG.remove();
             this._dragRectangleG = null;
             this._dragRectangle = null;
+
+            // Performance improvement: we could cache the full extend when addData() is called
+            let fullExtent = this._calculateFullExtent(this._areasFlattended);
+            fullExtent && this._map.fitBounds(fullExtent);
```

Comment:
Ask the map to zoom out so that the full route is visible.

Reply:
Could explain what this change does?

Reply:
BTW: yes I think caching would not hurt :+1:

Reply:
See https://mariusschulz.com/blog/the-and-and-or-operators-in-javascript#using-logical-operators-with-non-boolean-values

In this case, it executes the statement if and only if fullExtent is defined and not null (or any other value which would cast to a boolean false).

Reply:
I can read and write js :). Why is the change added? What problem does it solve?

Reply:
Oh, my bad then.

On selecting a subsection of the graph, the map zooms in or out to show only that portion of the route. On resetting the selection (aka the drag), the map should zoom out to show the full route.

Reply:
This is the equivalent of `fitBounds()` call to the map in the `_fitSection` function below.